src/shared/Git-Credential-Manager/Program.cs

```diff
@@ -29,5 +32,24 @@ public static void Main(string[] args)
                 Environment.Exit(exitCode);
             }
         }
+
+        private static string GetApplicationPath()
+        {
+            Assembly entryAssembly = Assembly.GetExecutingAssembly();
```

Comment:
Would any of these work as shorter options for the path?

1. Application.startupPath
2. System.IO.Path.GetDirectoryName( System.Reflection.Assembly.GetExecutingAssembly().Location)

Reply:
Neither of those include the filename, which I believe is the intent of the method.

Reply:
> Neither of those include the filename, which I believe is the intent of the method.

@jrbriggs is correct, but also..

> Application.startupPath

This is [a WinForms type](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.application.startuppath?view=netframework-4.8); we're a console application.